gprofiler/merge.py

```diff
@@ -209,8 +209,8 @@ def _parse_perf_script(script: Optional[str]) -> ProcessToStackSampleCounters:
     return pid_to_collapsed_stacks_counters
 
 
-def _make_profile_metadata(docker_client: Optional[DockerClient]) -> str:
-    if docker_client is not None:
+def _make_profile_metadata(docker_client: Optional[DockerClient], add_container_names: bool) -> str:
+    if docker_client is not None and add_container_names:
```

Comment:
```suggestion
    if add_container_names docker_client is not None:
```

for consistency with the other check u do in this file